Tasks/NuGet/nuget.ts

```diff
@@ -0,0 +1,94 @@
+import * as ngToolRunner from "nuget-task-common/NuGetToolRunner";
+import * as nutil from "nuget-task-common/Utility";
+import * as tl from "vsts-task-lib/task";
+global['_vsts_task_lib_loaded'] = true;
```

Comment:
add a comment here,since this can be removed once we're caught up to the task library version with the fix.